gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/service/MinioHandler.java

```diff
@@ -120,26 +131,27 @@ public void updateTasks(Event event) {
                             addWaitingFileAndNotifyTasks(processFileMinio, runningOrPendingTasks);
                         } else {
                             processFile = processFileRepository.save(processFile);
-                            Set<TaskWithStatusUpdate> taskWithStatusUpdates = taskService.addProcessFileToTasks(processFile, processFileMinio.getFileEventType(), true, true);
+                            Set<TaskWithStatusUpdate> taskWithStatusUpdates = taskService.addProcessFileToTasks(processFile,
+                                                                                                                processFileMinio.getFileEventType(), true, true);
```

Comment:
If you add a return carriage here, do it also with other parameters of the method. Here this is quite confusing.